app/src/main/java/com/wireguard/android/fragment/TunnelListFragment.kt

```diff
@@ -313,7 +377,7 @@ class TunnelListFragment : BaseFragment() {
 
     override fun onViewStateRestored(savedInstanceState: Bundle?) {
         super.onViewStateRestored(savedInstanceState)
-
+        setHasOptionsMenu(true)
```

Comment:
Why's this required?

Reply:
onCreateOptionsMenu only work in fragment when this is set, It is also set in TunnelDetailFragment.

Reply:
Ah yes, my bad.